### ISSUE TITLE
TypeError: Cannot read property 'value' of undefined

### ISSUE THREAD

**Issue by ramoncaldeira** [avocode/avocode#1138]
IMPORTANT: Please try to describe how did it happen.
I click on a rounded rectangle with a gradient background and the software breaks. After that, every click generates an error. Repeated multiple times.
-----
App v. `2.11.0`
Source UID: 56406
```
TypeError: Cannot read property 'value' of undefined
    at file:///usr/local/bin/avocode/resources/app.asar/dist/avocode.min.js:1:29713
    at Array.map (native)
    at t.value (file:///usr/local/bin/avocode/resources/app.asar/dist/avocode.min.js:1:29691)
    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (/usr/local/bin/avocode/resources/app.asar/node_modules/react/lib/ReactCompositeComponent.js:587:34)
    at ReactCompositeComponentWrapper._renderValidatedComponent (/usr/local/bin/avocode/resources/app.asar/node_modules/react/lib/ReactCompositeComponent.js:607:32)
    at ReactCompositeComponentWrapper.mountComponent (/usr/local/bin/avocode/resources/app.asar/node_modules/react/lib/ReactCompositeComponent.js:220:30)
    at Object.mountComponent (/usr/local/bin/avocode/resources/app.asar/node_modules/react/lib/ReactReconciler.js:37:35)
    at ReactDOMComponent.mountChildren (/usr/local/bin/avocode/resources/app.asar/node_modules/react/lib/ReactMultiCh
```

**Comment by jakubzitny** [comment]
Hi @ramoncaldeira, thanks for reporting this issue. This is a bug in latest version of Avocode, we've already prepared a fix for the next patch release.

Feel free to let us know if you have any more problems with Avocode 😉

**Comment by jakubzitny** [comment]
Hi, this has been fixed in 2.11.1 so I'm going to close this issue.

Feel free to reopen if you still have problems with this or anything else 😉
